spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1430,6 +1430,33 @@ class CometExecSuite extends CometTestBase {
       }
     })
   }
+
+  test("Windows support") {
+    Seq("true", "false").foreach(aqeEnabled =>
+      withSQLConf(
+        CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
+        SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> aqeEnabled) {
+        withParquetTable((0 until 10).map(i => (i, 10 - i)), "t1") { // TODO: test nulls
+          val aggregateFunctions =
+            List("COUNT(_1)", "MAX(_1)", "MIN(_1)") // TODO: Test all the aggregates
```

Comment:
Followup to test all the aggregates functions for window clause

Reply:
https://github.com/apache/datafusion-comet/issues/608

Reply:
Still have problems with other aggregates functions. I will look one by one.
For example:
```
sum: Error from DataFusion: Internal error: Builtin Sum will be removed
avg: org.apache.comet.CometNativeException: AvgAccumulator for (Int32 --> Float64)
```